### DISCUSSION TITLE
using github action when python project is in a subdir of the repo

### DISCUSSION THREAD

**Discussion by antony-amdl** [callowayproject/bump-my-version#263] [Q&A]
Trying to use `callowayproject/bump-my-version@master`

I am not sure how to point it to a subdir of the repo in the action yaml file 
(`working-directory` stuff is appicable for shell run steps only)

I see no arg to bump version for specifying a subdir.

**Comment by coordt** [comment]
I'll need a bit more information about your repo to answer. Bump-my-version is typically run from the repository's root and changes the files based on the configuration file in that root.

How is your repository structured?

**Comment by antony-amdl** [comment]
It's kind of a monorepo, so I needed to look into a subdir.
But, in any case i've found and decided to use `poetry version` to do the bumping instead since I'm already using poetry.
Thanks for getting back.
